clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/BulkUpsertEntityTagsAtomicOperation.java

```diff
@@ -71,7 +71,12 @@ public BulkUpsertEntityTagsAtomicOperationResult operate(List priorOutputs) {
       Map<String, EntityTags> existingTags = retrieveExistingTags(tags);
 
       getTask().updateStatus(BASE_PHASE, "Merging existing tags and metadata");
-      tags.forEach(tag -> mergeExistingTagsAndMetadata(now, existingTags.get(tag.getId()), tag, bulkUpsertEntityTagsDescription.isPartial));
+      tags.forEach(tag -> mergeExistingTagsAndMetadata(
```

Comment:
no changes here ... just took it upon myself to split an overly long line of code